### PR TITLE
Add operator manifest apply test

### PR DESCRIPTION
**Pull request by richardwxn**
depend on https://github.com/istio/operator/pull/246/files

### CODE REVIEW

File: prow/cluster/jobs/istio/operator/istio.operator.yaml [hunk 1]
```diff
@@ -77,3 +77,41 @@ presubmits:
...
+    spec:
+      containers:
+        - image: gcr.io/istio-testing/istio-builder:v20190829-4edc0e70
+          command:
+            - entrypoint
+            - prow/e2e_install.sh
+          env:
+            - name: GO111MODULE
+              value: "on"
+          securityContext:
```
Comment (howardjohn) [inline prow/cluster/jobs/istio/operator/istio.operator.yaml:99]:
```suggestion
          securityContext:
          resources:
            limits:
              cpu: "3"
              memory: 24Gi
            requests:
              cpu: 500m
              memory: 3Gi
```

**Comment by richardwxn** [comment]
Tested it locally and it worked fine

File: prow/cluster/jobs/istio/operator/istio.operator.yaml [hunk 2]
```diff
@@ -77,3 +77,48 @@ presubmits:
...
+          resources:
+            limits:
+              cpu: "3"
+              memory: 24Gi
+            requests:
+              cpu: 500m
+              memory: 3Gi
+            privileged: true
```
Comment (howardjohn) [inline prow/cluster/jobs/istio/operator/istio.operator.yaml]: this should be under security context not resources

Comment (richardwxn) [reply prow/cluster/jobs/istio/operator/istio.operator.yaml]: done

**Review by howardjohn — APPROVED** [review]

**Pull request merged**